archive/b/beef/testinfo.yml

```diff
@@ -4,7 +4,7 @@ folder:
 
 container:
   image: "rzuckerm/beef"
-  tag: "0.43.5a6-8"
```

Comment:
I just want to verify the version before I merge this one.

Reply:
This should be `0.43.5a6-8`. I just pushed a new commit to fix that.